cvmfs/shrinkwrap/shrinkwrap.cc

```diff
@@ -94,7 +94,7 @@ void Usage() {
         " -z --dest-config Dest config\n"
         " -t --spec-file   Specification file [default=$REPO.spec]\n"
         " -j --threads     Number of concurrent copy threads [default:2*CPUs]\n"
-        " -r --retries     Number of retries on copying file [default:0]\n"
+        " -p --stat-period Frequency of stat prints, 0 disables [default:10]\n"
```

Comment:
Why are we removing the `retries` option? I guess retry on download is covered by cvmfs options?

Reply:
At the time of removal this option did not actually impact retries. I can add this back in and have it passed through to the CVMFS options. It was intended for setting retries in shrinkwrap, but as the CVMFS client handled this its redundant and could cause excessive retries if CVMFS does a full round of retries for each shrinkwrap retry.

Reply:
I see, thanks!  Let's remove the `-r` option then.